docs/sql-guide/expressions/expressions-home.md

```diff
@@ -1,42 +1,83 @@
-title: SQL Expressions
+title: Expressions
 layout: default
 parent: SQL guide
-has_children: true
-nav_order: 5
-has_toc: false
 ---
 
 # SQL Expressions
+{: .no_toc }
+
+{% include page-toc.md %}
+
+## Before you begin
+
+{% include /cloud/cloud-db-create-before-begin.md %}
 
 ## identifier
 ![expr](/assets/images/sql-guide/identifier.svg)
 
+{% include /concepts/standard-naming-obj.md %}
+
 ## expr
+
 ![expr](/assets/images/sql-guide/expr.svg)
 
-## integer_literal
+{: .note}
+View SQL for [CREATE TABLE products and sales](/docs/sql-guide/statements/statement-table-create/#create-table-products-and-services-with-2-columns-and-constraint) and [INSERT INTO tables and products](/docs/sql-guide/statements/statement-insert-replace/#insert-multiple-records-into-products-and-services-table)
+
+| Syntax | Example | Result |
+|---|---|---|
+| `expr [NOT] LIKE expr` | `SELECT * FROM products WHERE prodlist LIKE 'pen';` | `ballpen`, `pencil`, `playpen` |
+| `expr IS [NOT] null` | `SELECT * FROM services WHERE price IS null;` | `free delivery on orders over $50` |
+| `expr [NOT] between expr AND expr` | `SELECT * from Products WHERE Price BETWEEN 10 AND 20;` | `fountain pen`, `notepad`, `rubber ball` |
+| `expr [NOT] IN ([SELECT_stmnt | expr,...])` | `Select * from Products where product_ID NOT IN (SELECT product_ID from Sales where product_name LIKE ear)` | `gold-plated earplugs` |
+
+### Literals
+
+Literals are explicitly specified fixed values that conform to a specific data type:
 
+| String literal | BNF diagram | Further information |
+|---|---|---|
+| `date` |  | [TIMESTAMP() data type](/docs/sql-guide/data-types/data-type-timestamp) |
+| `decimal` |  | [DECIMAL() data type](/docs/sql-guide/data-types/data-type-decimal)
+| `set` | ![expr](/assets/images/sql-guide/set_literal.svg) | [IDSET data type](/docs/sql-guide/data-types/data-type-idset)<br/>[STRINGSET data type](/docs/sql-guide/data-types/data-type-stringset) |
+| `string` |  | [STRING data type](/docs/sql-guide/data-types/data-type-string) |
+| `tuple` | ![expr](/assets/images/sql-guide/tuple_literal.svg) | Tuple literals are a collection of data types. |
 
-## string_literal
+### [table_name].column_name
 
 
-## decimal_literal
 
+### [unary_op] expr
 
-## set_literal
-![expr](/assets/images/sql-guide/set_literal.svg)
+* [Unary operators](/docs/sql-guide/operators/operators-home/#unary_op)
 
-## tuple_literal
-![expr](/assets/images/sql-guide/tuple_literal.svg)
+### expr binary_op expr
 
-## date_literal
+* [Binary operators](/docs/sql-guide/operators/operators-home/#binary_op)
+
+### function_call
 
-## function_call
 ![expr](/assets/images/sql-guide/function_call.svg)
 
-## paren_select_stmt
+* [SQL Functions](/docs/sql-guide/functions/functions-home)
+
+### CAST expr AS type_name
+
+| Syntax | Example | Result |
+|---|---|---|
+| `CAST expr AS type_name` | `SELECT CAST (25.65 AS int);` | 25 |
+
+* [CAST function](https://www.w3schools.com/sql/func_sqlserver_cast.asp){:target="_blank"}
```

Comment:
The reaper?!
W3 links are just to give existing/standard background info on the content (rather than reinventing the wheel). 
The logic is we fully document our implementation of SQL then refer users out to accurate information on **supported** SQL
W3schools was the simplest approach I could come up with. But if you've got something better, I'll use that instead :D
Back with a former employer we referred people to SQL-92 docs which looked a lot like photocopies converted to PDFs stored on a really old website. They were a bugger to navigate. This is better :D

Reply:
yup was just a warning. keep em